### DISCUSSION TITLE
新版本又出现闪退问题

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#607] [Q&A]
6约18号更新了版本7.8.6 (20240606.001718)，
然后就出现闪退情况，重新安装7.8.4后，情况仍然存在，不断重复地闪退闪退闪退......

**Comment by MaxMKMax** [comment]
@kidokogills 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by kidokogills** [reply]
73684827

[logs.zip](https://github.com/user-attachments/files/15890303/logs.zip)

**Reply by kidokogills** [reply]
主要是，这样情况，我能更新新版本吗？

**Reply by kidokogills** [reply]
目前情况是，系统win11，7.8.6，以及7.8.4版本都会出现闪退，Flutter 版无法升级至pro (pin码没反应，邮箱点击卡死)，该如何弄？ @Mike777Khoo

**Reply by netfriendBO** [reply]
确实是这样，我跟你情况一模一样

**Reply by MaxMKMax** [reply]
@netfriendBO 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by netfriendBO** [reply]
我发现，不联网时可以正常打开，只要连上网络就会闪退

**Reply by xiaolin6456** [reply]
> @netfriendBO 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
> 
> go to Support (转到支持) select Report an Issue (选择报告问题) choose issue type (选择问题类型) include additional details (包括额外的详细信息) hit Send Report (点击发送报告)
> 
> 当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!
> 
> 有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by netfriendBO** [comment]
73716516



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月19日(周三) 晚上11:42
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 新版本又出现闪退问题 (Discussion #607)




 
@netfriendBO 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
 
go to Support (转到支持)
 select Report an Issue (选择报告问题)
 choose issue type (选择问题类型)
 include additional details (包括额外的详细信息)
 hit Send Report (点击发送报告)
 
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!
 
有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by xiaolin6456** [reply]
不是不提交报告根本无法提交

**Reply by xiaolin6456** [reply]
现在很多人都遇到闪退问题，是个共性问题，团队无法修复吗，还是正在修复

**Comment by ycjdixon** [comment]
同样情况，不知道是什么原因？

**Reply by MaxMKMax** [reply]
@ycjdixon 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

**Comment by ycjdixon** [comment]
@kidokogills 遇到和你一样的情况。你可以试试退回7.8.4或者7.8.5版本，要先删除蓝灯再安装哦，你覆盖安装的话是不行的。我是这样操作的，不会闪退了。

**Reply by kidokogills** [reply]
现在只能用回7.8.4了，不稳定，重复点开还是有几率不闪退的，说来还有个神奇的点，就是下载的文件名里没标版本号，每次都是安装完，进页面才能看，就挺......麻烦的

**Comment by MaxMKMax** [comment]
@kidokogills @netfriendBO 
嗯嗯. 好的. 已经帮你们提交上去给技术部门了. 我们会尽快修复的. 谢谢您们耐心的等待和提供我们Log.

**Reply by kidokogills** [reply]
所以到底能不能，什么时候能够修复，到今天，7.8.6依然闪退

**Reply by MaxMKMax** [reply]
@kidokogills 可以修复. 但需要您提供我报告票号. 有了票号才方便 raise ticket. 我需要拿票号和技术团队谈. 谢谢您的耐心等待.

**Reply by GDAVKA** [reply]
问题是进去1秒就闪退了，根本没时间点到提交报告那里

**Reply by kidokogills** [reply]
@Mike777Khoo 哇，就是说，这段时间，半个月，没有主动排查，没有任何调整，什么都没做？过分了吧

**Reply by ycjdixon** [reply]
@kidokogills 
你总结到位

**Reply by MaxMKMax** [reply]
@GDAVKA @kidokogills 各位好. 可以帮我们提交一个应用内报告吗? 或者用手动方法给我们log. 谢谢. 明天开会我帮你们反馈问题给技术团队, 尽快修复. 麻烦你们啦. 谢谢帮忙.

**Comment by lemonskkyt** [comment]
测试版授权专业版不生效，正式版又一直闪退.....两个你倒是挑一个解决啊

**Reply by ycjdixon** [reply]
蓝灯团队已经换人了。

**Comment by MaxMKMax** [comment]
各位好. 我已经不断的反馈了. 再给我们一点点时间. 如果可以提供我log会更好. 谢谢啦!

**Reply by ycjdixon** [reply]
7.8.6版本改版了吗？闪退不能用。

**Comment by ycjdixon** [comment]
@Mike777Khoo  你们的技术团队还在工作吗？7.8.6版本闪退问题一直没有修复。安装完7.8.6版本后可以使用一些时间（时间长短不一：几十分钟到几小时都有），期间遇到连不上网后就自动退出软件了。之后再打开蓝灯就马上闪退了，一直闪退。还有8.0.0测试版也一直用不了。因为现在7.8.5版本已经很难联网了。蓝灯又处于不能用的时刻了。我们可是付费用户啊。用点心好吗！

**Reply by MaxMKMax** [reply]
@ycjdixon 您好. 非常抱歉影响了您的体验感. 您那里能帮我提交报告然后给我 8 票号.吗? 如果应用闪退, 提交不了, 您可以手动帮我找log然后提供给我吗? 非常感谢! 有log我们就能马上修复了. 麻烦您帮忙了. 谢谢! 
可以是看这么找:
C:\Users\username\AppData\Roaming\Lantern

**Comment by helanhao** [comment]
闪退的解决办法是：安装win7版本的就可以了，安装之后它会自动升级到7.8.4版本，不会闪退了

**Comment by MaxMKMax** [comment]
各位好. 可以帮我们提交一个应用内报告吗? 或者用手动方法给我们log. 谢谢. 明天开会我帮你们反馈问题给技术团队, 尽快修复. 麻烦你们啦. 谢谢帮忙.

**Comment by ycjdixon** [comment]
@Mike777Khoo 最新版本7.8.6闪退问题什么时候解决？测试版又不能用，你们团队就是不工作的吗？按理说这样的工作团队是应该时刻在线监测修复的啊！
